minheap-internal.h

```diff
@@ -81,8 +81,8 @@ struct event* min_heap_pop_(min_heap_t* s)
 	if (s->n)
 	{
 		struct event* e = *s->p;
-		min_heap_shift_down_(s, 0u, s->p[--s->n]);
-		e->ev_timeout_pos.min_heap_idx = -1;
+		min_heap_shift_down_(s, 0, s->p[--s->n]);
+		e->ev_timeout_pos.min_heap_idx = SIZE_MAX;
```

Comment:
`EV_SIZE_MAX` should be used instead

Reply:
Replaced SIZE_MAX occurrences with EV_SIZE_MAX.